addons/hr_expense/__openerp__.py

```diff
@@ -39,6 +39,7 @@
         'views/hr_dashboard.xml',
         'views/hr_expense.xml',
         'views/tour_views.xml',
+        'views/res_config_view.xml',
```

Comment:
should be `views/res_config_views.xml` (views is plural). Old habit kept in new guidelines ....